src/lib/progress-spinner/progress-spinner.scss

```diff
@@ -33,8 +33,8 @@ $mat-progress-spinner-viewport-size: 100px !default;
   path {
     fill: transparent;
 
-    // Stroke width of 10px defines stroke as 10% of the viewBox.
-    stroke-width: $mat-progress-spinner-stroke-width;
+    // Stroke width is by default 10px, which is relatively to the viewBox size 10%.
```

Comment:
That's a good point. In theory we could cache the `pathRadius` when the stroke width is updated through a `setter`. But I feel like it's not worth passing a cached value to the `getSvgArc` function (due to the trivial subtraction).

Let me know what you think. Happy to implement the caching if desired.